File: 프로그래머스/0/120893. 대문자와 소문자/대문자와 소문자.java
class Solution {
    public String solution(String my_string) {
        String answer = "";
        for (int i = 0; i < my_string.length(); i++) {
            char currentChar = my_string.charAt(i);
            if (Character.isUpperCase(currentChar)) {
                answer += Character.toLowerCase(currentChar);
            } else {
                answer += Character.toUpperCase(currentChar);
            }
        }
        return answer;
    }
}

File: 프로그래머스/0/120893. 대문자와 소문자/대문자와 소문자.py
def solution(my_string):
    answer = ''
    for str in range(len(my_string)):
        if my_string[str].isupper():
            answer = answer + my_string[str].lower()
        elif my_string[str].islower():
            answer = answer + my_string[str].upper()
            
    return answer